akka-actor-tests/src/test/scala/akka/io/dns/AsyncDnsResolverIntegrationSpec.scala

```diff
@@ -152,6 +154,24 @@ class AsyncDnsResolverIntegrationSpec extends AkkaSpec(
 
   }
 }
+
 object AsyncDnsResolverIntegrationSpec {
-  lazy val dockerDnsServerPort = SocketUtil.temporaryLocalPort()
+  lazy val dockerDnsServerPort: Int = {
+    def freePort(tries: Int): Int = {
```

Comment:
Sounds like something that would be useful in `SocketUtil`, perhaps move it?

Reply:
I agree, note that it has some stuff for RANDOM_LOOPBACK_ADDRESS